framework/src/media/audio/audio_manager.c

```diff
@@ -1247,56 +1251,48 @@ uint32_t get_output_bytes_to_frame(unsigned int bytes)
 
 audio_manager_result_t get_max_audio_volume(uint8_t *volume)
 {
-	// ToDo: Add logics to consider 'decibel'.
```

Comment:
we check in player already but it is different layer, we'd better keep it here.

Reply:
Okay, reverted.